junwoo/controllers/likeController.js

```diff
@@ -0,0 +1,11 @@
+const likeService = require("../services/likeService");
+
+const likeUp = async (req, res) => {
```

Comment:
함수명이 조금 애매한 것 같습니다.
특정 행위와 결과를 예측할 수 있도록 함수명을 수정해주세요!
```suggestion
const createLike = async (req, res) => {
```